src/gui/optionsdlg.ui

```diff
@@ -1581,14 +1581,28 @@
                 </widget>
                </item>
                <item>
-                <widget class="QCheckBox" name="checkForceProxy">
-                 <property name="text">
-                  <string>Disable connections not supported by proxies</string>
-                 </property>
-                 <property name="checked">
-                  <bool>true</bool>
-                 </property>
-                </widget>
+                <layout class="QHBoxLayout" name="forceProxyHorizontalLayout">
+                 <item>
+                  <widget class="QCheckBox" name="checkForceProxy">
+                   <property name="text">
+                    <string>Disable connections not supported by proxies</string>
+                   </property>
+                   <property name="checked">
+                    <bool>true</bool>
+                   </property>
+                  </widget>
+                 </item>
+                 <item>
+                  <widget class="QLabel" name="labelForceProxy">
+                   <property name="text">
+                    <string> (&lt;a href=&quot;https://github.com/qbittorrent/qBittorrent/wiki/Disable-connections-not-supported-by-proxies&quot;&gt;More information&lt;/a&gt;)</string>
```

Comment:
is the space at the start of the string intended?

Reply:
The "More information" instance used for Anonymous Mode prepends a space, so I did the same for consistency. I can remove from both if desired.